A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -460,6 +480,164 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
+    getManagedTargets(v, m) {
+        const vM = _convertMachToKCas(m, _convertCtoK(Simplane.getAmbientTemperature()), SimVar.GetSimVarValue("AMBIENT PRESSURE", "millibar"));
+        return v > vM ? [vM, true] : [v, false];
+    }
+
+    updateManagedSpeeds() {
+        if (!this.managedSpeedClimbIsPilotEntered) {
+            this.managedSpeedClimb = this.getClbManagedSpeedFromCostIndex();
+        }
+        if (!this.managedSpeedCruiseIsPilotEntered) {
+            this.managedSpeedCruise = this.getCrzManagedSpeedFromCostIndex();
+        }
+        if (!this.managedSpeedDescendIsPilotEntered) {
+            this.managedSpeedDescend = this.getDesManagedSpeedFromCostIndex();
+        }
+    }
+
+    updateManagedSpeed() {
+        if (!this.isAirspeedManaged()) {
+            return;
+        }
+
+        let vPfd = 0;
+        let isMach = false;
+
+        if (SimVar.GetSimVarValue("L:A32NX_FMA_EXPEDITE_MODE", "number") === 1) {
+            const verticalMode = SimVar.GetSimVarValue("L:A32NX_FMA_VERTICAL_MODE", "number");
+            if (verticalMode === 12) {
```

Comment:
That's one of the things being open, I plan to document all local variables. This includes name, purpose and possible values and their meaning. This will resolve the above from my point of view.

Reply:
I added now the documentation of the variables to the branch.